exercises/dnd-character/README.md

```diff
@@ -0,0 +1,93 @@
+# Dnd Character
```

Comment:
oh right, I forgot. You should add a `title` to https://github.com/exercism/problem-specifications/blob/master/exercises/dnd-character/metadata.yml if you want the first word to show up as something other than Dnd. Perhaps you want DnD or DND.

Reply:
Thanks, I just did that in exercism/problem-specifications#1417.